src/config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   host: process.env.host || "localhost",
-  login: process.env.login || "test",
-  password: process.env.password || "test",
+  login: process.env.login || "guest",
```

Comment:
We can probably ignore this as the default creds are test in docker-compose and in `.env.sample` file

Reply:
With ignore I mean revert the changes using
`git checkout origin/master -- src/config.hs`

Reply:
okay